ci/Jenkinsfile-linux-binary

```diff
@@ -43,11 +43,32 @@ def binaryBuilder = builder('py35', '/workspace')({
         }
     }
 
-    stage ("Build dcos-cli binary") {
-        dir('dcos-cli/python/lib/dcoscli') {
+    dir('dcos-cli/python/lib/dcoscli') {
+        stage ("Build dcos-cli binary") {
             sh "make binary"
             sh "dist/dcos"
         }
+
+        stage ("Run syntax checks on dcoscli") {
+            sh "make env"
+            sh '''
+                bash -c " \
+                  source env/bin/activate && \
+                  tox -e py35-syntax"
```

Comment:
I've left out the dcoscli unit tests as they unfortunately currently require a configured cluster.

Reply:
Do we have an open ticket regarding this? Unit tests should be able to run without a configured cluster in the future 🤔.

Reply:
Indeed :+1:  I've created https://jira.mesosphere.com/browse/DCOS_OSS-2270